SodaRu-Dev/battleship
# ls-1 - created index.html, js and scss directories, and gruntfile runs

created initial files and directories:
- index.html
- scripts/main.js
- styles/styles.scss
- gruntfile.js
- package.json
- README.md
- .gitignore

testing instructions:
1. fetch branch
2. follow installation instructions in README.md
3. ensure the following files are created in /dist/
- index.html
- styles/styles.css
- styles/styles.css.map
- scripts/bundle.js

closes #1
